Dockerfile

```diff
@@ -7,14 +7,11 @@ WORKDIR /community-server
 ## Copy the package.json for audit
 COPY package*.json ./
 
-## Verify if there are known vulnerabilities in the dependencies
-RUN npm audit --production --audit-level=high
-
 ## Copy the dockerfile's context's community server files
 COPY . .
 
 ## Install and build the Solid community server (prepare script cannot run in wd)
-RUN npm ci --ignore-scripts && npm run build
+RUN npm ci && npm run build
```

Comment:
```suggestion
RUN npm ci --unsafe-perm && npm run build
```

I think we might need to add back the `--unsafe-perm` flag now that we removed `--ignore-scripts`.

See: https://github.com/solid/community-server/pull/580#issuecomment-780906324